pyproject.toml

```diff
@@ -10,7 +10,8 @@ requires = [
     # we don't want to force users to compile with 1.25 though
     # (Ideally, in the future, though, oldest-supported-numpy can be dropped when our min numpy is 1.25.x)
     "oldest-supported-numpy>=2022.8.16; python_version<'3.12'",
-    "numpy>=1.22.4; python_version>='3.12'",
+    # TODO: This needs to be updated when the official numpy 1.26 comes out
+    "numpy>=1.26.0b1; python_version>='3.12'",
```

Comment:
Could we just use `numpy>1.25`?

Reply:
I don't know if pip will pull a pre-release without explicitly specifying this.

This should be nbd though, I'm not planning on publishing the built wheels to PyPI anyways until numpy 1.26 is out.